config/game_options.txt

```diff
@@ -1,519 +1,520 @@
+## Set to 2 for "random", silicons will start with a random lawset picked from random laws specified below.
+## Set to 3 for "weighted random", using values in "silicon_weights.txt", a law will be selected, with weights specifed in that file.
+DEFAULT_LAWS 0
+
+## RANDOM LAWS ##
+## ------------------------------------------------------------------------------------------
+## These control what laws are available for selection if random silicon laws are active.
+## See datums\ai_laws.dm for the full law lists
+
+## standard-ish laws. These are fairly ok to run
+RANDOM_LAWS asimov
+RANDOM_LAWS asimovpp
+RANDOM_LAWS paladin
+RANDOM_LAWS robocop
+RANDOM_LAWS corporate
+
+## Quirky laws. Shouldn't cause too much harm
+#RANDOM_LAWS hippocratic
+#RANDOM_LAWS maintain
+#RANDOM_LAWS drone
+#RANDOM_LAWS liveandletlive
+#RANDOM_LAWS peacekeeper
+#RANDOM_LAWS reporter
+
+## Bad idea laws. Probably shouldn't enable these
+#RANDOM_LAWS syndie
+#RANDOM_LAWS ninja
+#RANDOM_LAWS antimov
+#RANDOM_LAWS thermodynamic
+#RANDOM_LAWS ratvar
+
+## meme laws. Honk
+#RANDOM_LAWS buildawall
+
+## If weighted laws are selected (DEFAULT_LAWS = 3),
+## then an AI's starting laws will be determined by the weights of these values
+
+## Make sure there are no spaces between the law_id and the number.
+
+LAW_WEIGHT custom,0
+
+## standard-ish laws. These are fairly ok to run
+LAW_WEIGHT asimov,32
+LAW_WEIGHT asimovpp,12
+LAW_WEIGHT paladin,12
+LAW_WEIGHT robocop,12
+LAW_WEIGHT corporate,12
+
+## Quirky laws. Shouldn't cause too much harm
+LAW_WEIGHT hippocratic,3
+LAW_WEIGHT maintain,4
+LAW_WEIGHT drone,3
+LAW_WEIGHT liveandletlive,3
+LAW_WEIGHT peacekeeper,3
+LAW_WEIGHT reporter,4
+
+## Bad idea laws. Probably shouldn't enable these
+LAW_WEIGHT syndie,0
+LAW_WEIGHT ninja,0
+LAW_WEIGHT antimov,0
+LAW_WEIGHT thermodynamic,0
+LAW_WEIGHT ratvar,0
+LAW_WEIGHT buildawall,0
+
+##------------------------------------------------
+
+## SILICON LAW MAX AMOUNT ###
+## The maximum number of laws a silicon can have
+## Attempting to upload laws past this point will fail unless the AI is reset
+SILICON_MAX_LAW_AMOUNT 12
+
+## Uncomment to give players the choice of their species before they join the game
+JOIN_WITH_MUTANT_RACE
+
+
+## Roundstart Races
+##-------------------------------------------------------------------------------------------
+## Uncommenting races will allow them to be choosen at roundstart while join_with_muntant_race is on. You'll need at least one.
+
+## You probably want humans on your space station, but technically speaking you can turn them off without any ill effect
+ROUNDSTART_RACES human
+ROUNDSTART_RACES mammal
+ROUNDSTART_RACES avian
+ROUNDSTART_RACES aquatic
+ROUNDSTART_RACES insect
+ROUNDSTART_RACES xeno
+ROUNDSTART_RACES datashark
+ROUNDSTART_RACES guilmon
+ROUNDSTART_RACES ipc

```

Comment:
Revert changes to this file, try using notepad++ instead of whatever you used to edit this file, overly large diff is large

Reply:
turns out this staged the wrong file into the change, reeee